src/utils.ts

```diff
@@ -293,13 +302,13 @@ export async function getBrowserPath(config: Partial<IUserConfig> = {}): Promise
 
 /**
  * Gets a supported debug config and updates the status of the launch.json file associated with the current workspace
- * @returns {vscode.DebugConfiguration | null}
+ * @returns {vscode.DebugConfiguration | string}
  */
- export function getLaunchJson(): vscode.DebugConfiguration | null {
+ export function getLaunchJson(): vscode.DebugConfiguration | string {
```

Comment:
I don't love using different return types to indicate state, but I can see how we would need to.  Let us at least scope in what values are acceptable instead of a generic string return type.

I was thinking we could use a type and do something like this:
```
type LaunchConfig = 'None' | 'Unsupported' | vscode.DebugConfiguration;
...
export function getLaunchJson(): LaunchConfig { ...
```